### ISSUE TITLE
zigbee2MQTT AddOn upgrade broke with SiLabs-Multiprotocol AddOn

### ISSUE THREAD

**Issue by tommyjlong** [zigbee2mqtt/hassio-zigbee2mqtt#469]
### Description of the issue

(New user).  I was able to get zigbee2mqtt Add-On version 1.30.1-1 working with SiLabs-multiprotocol AddOn version 0.13.1 (w. SkyConnect).  z2m is connected to SiLabs-multiprotocol's Zigbee stack with a tcp socket and specifying ezsp as the adapter. 

After upgrading to version 1.30.2-1 this no longer working.  After downgrading back to 1.30.1 it works again.

### Addon version

v1.30.2-1

### Platform

Home Assistant 2023.3.0 (or 2023.2.4)
Supervisor 2023.01.1
Operating System 9.5
Frontend 20230301.0 - latest

### Logs of the issue (if applicable)

Here is the startup log:
```
[13:48:02] INFO: Preparing to start...
[13:48:02] INFO: Socat not enabled
[13:48:02] INFO: Zigbee Herdsman debug logging enabled
[13:48:03] INFO: Starting Zigbee2MQTT...
Zigbee2MQTT:debug 2023-03-03 13:48:06: Loaded state from file /config/zigbee2mqtt/state.json
Zigbee2MQTT:info  2023-03-03 13:48:06: Logging to console and directory: '/config/zigbee2mqtt/log/2023-03-03.13-48-06' filename: log.txt
Zigbee2MQTT:debug 2023-03-03 13:48:06: Removing old log directory '/config/zigbee2mqtt/log/2023-03-02.11-47-38'
Zigbee2MQTT:info  2023-03-03 13:48:06: Starting Zigbee2MQTT version 1.30.2 (commit #unknown)
Zigbee2MQTT:info  2023-03-03 13:48:06: Starting zigbee-herdsman (0.14.96)
Zigbee2MQTT:debug 2023-03-03 13:48:06: Using zigbee-herdsman with settings: '{"adapter":{"concurrent":null,"delay":null,"disableLED":false},"backupPath":"/config/zigbee2mqtt/coordinator_backup.json","databaseBackupPath":"/config/zigbee2mqtt/database.db.backup","databasePath":"/config/zigbee2mqtt/database.db","network":{"channelList":[25],"extendedPanID":[221,221,221,221,221,221,221,221],"networkKey":"HIDDEN","panID":6754},"serialPort":{"adapter":"ezsp","path":"tcp://core-silabs-multiprotocol:9999"}}'
2023-03-03T18:48:06.697Z zigbee-herdsman:controller:log Starting with options '{"network":{"networkKeyDistribute":false,"networkKey":[blah-blah],"panID":6754,"extendedPanID":[221,221,221,221,221,221,221,221],"channelList":[25]},"serialPort":{"path":"tcp://core-silabs-multiprotocol:9999","adapter":"ezsp"},"databasePath":"/config/zigbee2mqtt/database.db","databaseBackupPath":"/config/zigbee2mqtt/database.db.backup","backupPath":"/config/zigbee2mqtt/coordinator_backup.json","adapter":{"disableLED":false,"concurrent":null,"delay":null}}'
2023-03-03T18:48:06.700Z zigbee-herdsman:adapter:ezsp:uart Opening TCP socket with core-silabs-multiprotocol:9999
2023-03-03T18:48:06.716Z zigbee-herdsman:adapter:ezsp:uart Socket connected
2023-03-03T18:48:06.717Z zigbee-herdsman:adapter:ezsp:uart Socket ready
2023-03-03T18:48:06.717Z zigbee-herdsman:adapter:ezsp:uart Uart reseting
2023-03-03T18:48:06.718Z zigbee-herdsman:adapter:ezsp:uart --> Write reset
2023-03-03T18:48:06.722Z zigbee-herdsman:adapter:ezsp:uart --> [1ac038bc7e]
2023-03-03T18:48:06.729Z zigbee-herdsman:adapter:ezsp:uart -?- waiting reset
2023-03-03T18:48:06.736Z zigbee-herdsman:adapter:ezsp:uart <-- []
2023-03-03T18:48:06.737Z zigbee-herdsman:adapter:ezsp:uart <-- [c1]
2023-03-03T18:48:06.738Z zigbee-herdsman:adapter:ezsp:uart <-- [02]
2023-03-03T18:48:06.739Z zigbee-herdsman:adapter:ezsp:uart <-- [0b]
2023-03-03T18:48:06.740Z zigbee-herdsman:adapter:ezsp:uart <-- [0a527e]
2023-03-03T18:48:06.742Z zigbee-herdsman:adapter:ezsp:uart <-- RSTACK: c1020b0a527e
2023-03-03T18:48:06.743Z zigbee-herdsman:adapter:ezsp:uart RSTACK Version: 2 Reason: RESET_SOFTWARE frame: c1020b0a527e
2023-03-03T18:48:06.745Z zigbee-herdsman:adapter:ezsp:uart -+- waiting reset success
2023-03-03T18:48:06.746Z zigbee-herdsman:adapter:ezsp:ezsp ==> version: {"desiredProtocolVersion":4}
2023-03-03T18:48:06.749Z zigbee-herdsman:adapter:ezsp:ezsp ==> {"_cls_":"version","_id_":0,"_isRequest_":true,"desiredProtocolVersion":4}
2023-03-03T18:48:06.750Z zigbee-herdsman:adapter:ezsp:uart --> DATA (0,0,0): 00000004
2023-03-03T18:48:06.751Z zigbee-herdsman:adapter:ezsp:uart --> [004221a850ed2c7e]
2023-03-03T18:48:06.752Z zigbee-herdsman:adapter:ezsp:uart -?- waiting (1)
2023-03-03T18:48:08.752Z zigbee-herdsman:adapter:ezsp:uart --> Error: Error: {"sequence":1} after 2000ms
2023-03-03T18:48:08.753Z zigbee-herdsman:adapter:ezsp:uart -!- break waiting (1)
2023-03-03T18:48:08.754Z zigbee-herdsman:adapter:ezsp:uart Can't send DATA frame (0,0,0): 00000004
2023-03-03T18:48:09.255Z zigbee-herdsman:adapter:ezsp:uart ->> DATA (0,0,1): 00000004
2023-03-03T18:48:09.256Z zigbee-herdsman:adapter:ezsp:uart --> [084221a850ef017e]
2023-03-03T18:48:09.256Z zigbee-herdsman:adapter:ezsp:uart -?- rewaiting (1)
2023-03-03T18:48:11.052Z zigbee-herdsman:adapter:ezsp:uart <-- []
2023-03-03T18:48:11.054Z zigbee-herdsman:adapter:ezsp:uart <-- [c1]
2023-03-03T18:48:11.055Z zigbee-herdsman:adapter:ezsp:uart <-- [02]
2023-03-03T18:48:11.056Z zigbee-herdsman:adapter:ezsp:uart <-- [0b]
2023-03-03T18:48:11.058Z zigbee-herdsman:adapter:ezsp:uart <-- [0a]
2023-03-03T18:48:11.059Z zigbee-herdsman:adapter:ezsp:uart <-- [52]
2023-03-03T18:48:11.060Z zigbee-herdsman:adapter:ezsp:uart <-- [7e]
2023-03-03T18:48:11.060Z zigbee-herdsman:adapter:ezsp:uart <-- RSTACK: c1020b0a527e
2023-03-03T18:48:11.061Z zigbee-herdsman:adapter:ezsp:uart RSTACK Version: 2 Reason: RESET_SOFTWARE frame: c1020b0a527e
2023-03-03T18:48:11.065Z zigbee-herdsman:adapter:ezsp:uart <-- [01]
2023-03-03T18:48:11.065Z zigbee-herdsman:adapter:ezsp:uart <-- [42]
2023-03-03T18:48:11.066Z zigbee-herdsman:adapter:ezsp:uart <-- [a1]
2023-03-03T18:48:11.066Z zigbee-herdsman:adapter:ezsp:uart <-- [a8]
2023-03-03T18:48:11.067Z zigbee-herdsman:adapter:ezsp:uart <-- [5e]
2023-03-03T18:48:11.068Z zigbee-herdsman:adapter:ezsp:uart <-- [28]
2023-03-03T18:48:11.068Z zigbee-herdsman:adapter:ezsp:uart <-- [05]
2023-03-03T18:48:11.069Z zigbee-herdsman:adapter:ezsp:uart <-- [c0]
2023-03-03T18:48:11.070Z zigbee-herdsman:adapter:ezsp:uart <-- [999c7e]
2023-03-03T18:48:11.070Z zigbee-herdsman:adapter:ezsp:uart <-- DATA (0,1,0): 0142a1a85e2805c0999c7e
2023-03-03T18:48:11.071Z zigbee-herdsman:adapter:ezsp:uart --> ACK  (1)
2023-03-03T18:48:11.072Z zigbee-herdsman:adapter:ezsp:uart --> [8160597e]
2023-03-03T18:48:11.073Z zigbee-herdsman:adapter:ezsp:ezsp <== Frame: 0080000a021072
2023-03-03T18:48:11.074Z zigbee-herdsman:adapter:ezsp:ezsp <== 0x0: {"_cls_":"version","_id_":0,"_isRequest_":false,"protocolVersion":10,"stackType":2,"stackVersion":29200}
2023-03-03T18:48:11.075Z zigbee-herdsman:adapter:ezsp:uart -+- rewaiting (1) success
2023-03-03T18:48:11.076Z zigbee-herdsman:adapter:ezsp:uart -+- waiting (1) success
2023-03-03T18:48:11.076Z zigbee-herdsman:adapter:ezsp:ezsp Switching to eszp version 10
2023-03-03T18:48:11.076Z zigbee-herdsman:adapter:ezsp:ezsp ==> version: {"desiredProtocolVersion":10}
2023-03-03T18:48:11.077Z zigbee-herdsman:adapter:ezsp:ezsp ==> {"_cls_":"version","_id_":0,"_isRequest_":true,"desiredProtocolVersion":10}
2023-03-03T18:48:11.078Z zigbee-herdsman:adapter:ezsp:uart --> DATA (0,1,0): 01000100000a
2023-03-03T18:48:11.078Z zigbee-herdsman:adapter:ezsp:uart --> [014321a9542a1fde467e]
2023-03-03T18:48:11.079Z zigbee-herdsman:adapter:ezsp:uart -?- waiting (1)
2023-03-03T18:48:11.079Z zigbee-herdsman:adapter:ezsp:uart <-- [8160597e]
2023-03-03T18:48:11.080Z zigbee-herdsman:adapter:ezsp:uart <-- ACK  (1): 8160597e
2023-03-03T18:48:11.080Z zigbee-herdsman:adapter:ezsp:uart -+- waiting (1) success
2023-03-03T18:48:12.053Z zigbee-herdsman:adapter:ezsp:uart <-- [a1443b7e]
2023-03-03T18:48:12.054Z zigbee-herdsman:adapter:ezsp:uart <-- NAK  (1): a1443b7e
2023-03-03T18:48:12.055Z zigbee-herdsman:adapter:ezsp:uart NAK Unexpected packet sequence 1
2023-03-03T18:48:16.751Z zigbee-herdsman:adapter:ezsp:ezsp Time to watchdog ... 0
2023-03-03T18:48:16.751Z zigbee-herdsman:adapter:ezsp:ezsp ==> nop: null
2023-03-03T18:48:21.083Z zigbee-herdsman:adapter:ezsp:ezsp ==> {"_cls_":"nop","_id_":5,"_isRequest_":true}
2023-03-03T18:48:21.083Z zigbee-herdsman:adapter:ezsp:uart --> DATA (1,1,0): 0200010500
2023-03-03T18:48:21.084Z zigbee-herdsman:adapter:ezsp:uart --> [7d314021a9512a9a0d7e]
2023-03-03T18:48:21.088Z zigbee-herdsman:adapter:ezsp:uart -?- waiting (2)
Zigbee2MQTT:error 2023-03-03 13:48:21: Error while starting zigbee-herdsman
Zigbee2MQTT:error 2023-03-03 13:48:21: Failed to start zigbee
Zigbee2MQTT:error 2023-03-03 13:48:21: Check https://www.zigbee2mqtt.io/guide/installation/20_zigbee2mqtt-fails-to-start.html for possible solutions
Zigbee2MQTT:error 2023-03-03 13:48:21: Exiting...
Zigbee2MQTT:error 2023-03-03 13:48:21: Error: Failure send version:{"type":"Buffer","data":[1,0,1,0,0,10]}
    at /app/node_modules/zigbee-herdsman/src/adapter/ezsp/driver/ezsp.ts:551:23
    at Queue.executeNext (/app/node_modules/zigbee-herdsman/src/utils/queue.ts:32:32)
2023-03-03T18:48:21.156Z zigbee-herdsman:adapter:ezsp:uart <-- [1240a1a90c2a25bdf97e]
2023-03-03T18:48:21.156Z zigbee-herdsman:adapter:ezsp:uart <-- DATA (1,2,0): 1240a1a90c2a25bdf97e
2023-03-03T18:48:21.156Z zigbee-herdsman:adapter:ezsp:uart --> ACK  (2)
2023-03-03T18:48:21.157Z zigbee-herdsman:adapter:ezsp:uart --> [82503a7e]
2023-03-03T18:48:21.157Z zigbee-herdsman:adapter:ezsp:ezsp <== Frame: 028001580030
2023-03-03T18:48:21.158Z zigbee-herdsman:adapter:ezsp:ezsp <== 0x58: {"_cls_":"invalidCommand","_id_":88,"_isRequest_":false,"reason":48}
2023-03-03T18:48:21.158Z zigbee-herdsman:adapter:ezsp:uart -+- waiting (2) success
```

ALSO for comparison, here is a similar log when working:

```
[19:49:48] INFO: [32mPreparing to start...[0m
[19:49:48] INFO: [32mSocat not enabled[0m
[19:49:48] INFO: [32mZigbee Herdsman debug logging enabled[0m
[19:49:49] INFO: [32mStarting Zigbee2MQTT...[0m
Zigbee2MQTT:debug 2023-03-01 19:49:52: Loaded state from file /config/zigbee2mqtt/state.json
Zigbee2MQTT:info  2023-03-01 19:49:52: Logging to console and directory: '/config/zigbee2mqtt/log/2023-03-01.19-49-52' filename: log.txt
Zigbee2MQTT:debug 2023-03-01 19:49:52: Removing old log directory '/config/zigbee2mqtt/log/2023-03-01.14-03-43'
Zigbee2MQTT:info  2023-03-01 19:49:52: Starting Zigbee2MQTT version 1.30.1 (commit #unknown)
Zigbee2MQTT:info  2023-03-01 19:49:52: Starting zigbee-herdsman (0.14.89)
Zigbee2MQTT:debug 2023-03-01 19:49:52: Using zigbee-herdsman with settings: '{"adapter":{"concurrent":null,"delay":null,"disableLED":false},"backupPath":"/config/zigbee2mqtt/coordinator_backup.json","databaseBackupPath":"/config/zigbee2mqtt/database.db.backup","databasePath":"/config/zigbee2mqtt/database.db","network":{"channelList":[25],"extendedPanID":[221,221,221,221,221,221,221,221],"networkKey":"HIDDEN","panID":6754},"serialPort":{"adapter":"ezsp","path":"tcp://core-silabs-multiprotocol:9999"}}'
2023-03-02T00:49:52.641Z zigbee-herdsman:controller:log Starting with options '{"network":{"networkKeyDistribute":false,"networkKey":[blah-blah],"panID":6754,"extendedPanID":[221,221,221,221,221,221,221,221],"channelList":[25]},"serialPort":{"path":"tcp://core-silabs-multiprotocol:9999","adapter":"ezsp"},"databasePath":"/config/zigbee2mqtt/database.db","databaseBackupPath":"/config/zigbee2mqtt/database.db.backup","backupPath":"/config/zigbee2mqtt/coordinator_backup.json","adapter":{"disableLED":false,"concurrent":null,"delay":null}}'
2023-03-02T00:49:52.655Z zigbee-herdsman:adapter:ezsp:uart Opening TCP socket with core-silabs-multiprotocol:9999
2023-03-02T00:49:52.681Z zigbee-herdsman:adapter:ezsp:uart Socket connected
2023-03-02T00:49:52.687Z zigbee-herdsman:adapter:ezsp:uart Socket ready
2023-03-02T00:49:52.687Z zigbee-herdsman:adapter:ezsp:uart Uart reseting
2023-03-02T00:49:52.687Z zigbee-herdsman:adapter:ezsp:uart --> Write reset
2023-03-02T00:49:52.688Z zigbee-herdsman:adapter:ezsp:uart --> [1ac038bc7e]
2023-03-02T00:49:52.692Z zigbee-herdsman:adapter:ezsp:uart <-- [007d5db1a96b2a164fa674eb04aaa76d499db3d844e7ce6794da7e]
2023-03-02T00:49:52.693Z zigbee-herdsman:adapter:ezsp:uart <-- DATA (0,0,0): 007db1a96b2a164fa674eb04aaa76d499db3d844e7ce6794da7e
2023-03-02T00:49:52.694Z zigbee-herdsman:adapter:ezsp:uart --> ACK  (1)
2023-03-02T00:49:52.694Z zigbee-herdsman:adapter:ezsp:uart --> [8160597e]
2023-03-02T00:49:52.714Z zigbee-herdsman:adapter:ezsp:ezsp <== Frame: 3f90013f0003fdffe0a12100f2ff0001fdffef0a0000
2023-03-02T00:49:52.716Z zigbee-herdsman:adapter:ezsp:ezsp <== 0x1: 
{"_id_":1,"_cls_":"getLibraryStatus","_isRequest_":false,"status":63}
2023-03-02T00:49:57.093Z zigbee-herdsman:adapter:ezsp:uart <-- []
2023-03-02T00:49:57.094Z zigbee-herdsman:adapter:ezsp:uart <-- [c1]
2023-03-02T00:49:57.096Z zigbee-herdsman:adapter:ezsp:uart <-- [02]
2023-03-02T00:49:57.097Z zigbee-herdsman:adapter:ezsp:uart <-- [0b]
2023-03-02T00:49:57.098Z zigbee-herdsman:adapter:ezsp:uart <-- [0a]
2023-03-02T00:49:57.098Z zigbee-herdsman:adapter:ezsp:uart <-- [52]
2023-03-02T00:49:57.099Z zigbee-herdsman:adapter:ezsp:uart <-- [7e]
2023-03-02T00:49:57.099Z zigbee-herdsman:adapter:ezsp:uart <-- RSTACK: c1020b0a527e
2023-03-02T00:49:57.100Z zigbee-herdsman:adapter:ezsp:uart RSTACK Version: 2 Reason: RESET_SOFTWARE frame: c1020b0a527e
2023-03-02T00:49:57.101Z zigbee-herdsman:adapter:ezsp:ezsp ==> version: {"desiredProtocolVersion":4}
2023-03-02T00:49:57.103Z zigbee-herdsman:adapter:ezsp:ezsp ==> {"_cls_":"version","_id_":0,"_isRequest_":true,"desiredProtocolVersion":4}
2023-03-02T00:49:57.106Z zigbee-herdsman:adapter:ezsp:uart --> DATA (0,0,0): 00000004
2023-03-02T00:49:57.112Z zigbee-herdsman:adapter:ezsp:uart --> [004221a850ed2c7e]
2023-03-02T00:49:57.113Z zigbee-herdsman:adapter:ezsp:uart -?- waiting (1)
2023-03-02T00:49:57.115Z zigbee-herdsman:adapter:ezsp:uart <-- [01]
2023-03-02T00:49:57.116Z zigbee-herdsman:adapter:ezsp:uart <-- [42]
2023-03-02T00:49:57.117Z zigbee-herdsman:adapter:ezsp:uart <-- [a1]
2023-03-02T00:49:57.117Z zigbee-herdsman:adapter:ezsp:uart <-- [a8]
2023-03-02T00:49:57.118Z zigbee-herdsman:adapter:ezsp:uart <-- [5e]
2023-03-02T00:49:57.119Z zigbee-herdsman:adapter:ezsp:uart <-- [2805c0999c7e]
2023-03-02T00:49:57.119Z zigbee-herdsman:adapter:ezsp:uart <-- DATA (0,1,0): 0142a1a85e2805c0999c7e
2023-03-02T00:49:57.119Z zigbee-herdsman:adapter:ezsp:uart --> ACK  (1)
2023-03-02T00:49:57.120Z zigbee-herdsman:adapter:ezsp:uart --> [8160597e]
2023-03-02T00:49:57.121Z zigbee-herdsman:adapter:ezsp:ezsp <== Frame: 0080000a021072
2023-03-02T00:49:57.122Z zigbee-herdsman:adapter:ezsp:ezsp <== 0x0: {"_id_":0,"_cls_":"version","_isRequest_":false,"protocolVersion":10,"stackType":2,"stackVersion":29200}
2023-03-02T00:49:57.122Z zigbee-herdsman:adapter:ezsp:uart -+- waiting (1) success
2023-03-02T00:49:57.123Z zigbee-herdsman:adapter:ezsp:ezsp Switching to eszp version 10
2023-03-02T00:49:57.124Z zigbee-herdsman:adapter:ezsp:ezsp ==> version: {"desiredProtocolVersion":10}
2023-03-02T00:49:57.124Z zigbee-herdsman:adapter:ezsp:ezsp ==> {"_cls_":"version","_id_":0,"_isRequest_":true,"desiredProtocolVersion":10}
2023-03-02T00:49:57.126Z zigbee-herdsman:adapter:ezsp:uart --> DATA (1,1,0): 01000100000a
2023-03-02T00:49:57.127Z zigbee-herdsman:adapter:ezsp:uart --> [7d314321a9542a1fe93d7e]
2023-03-02T00:49:57.128Z zigbee-herdsman:adapter:ezsp:uart -?- waiting (2)
2023-03-02T00:49:57.128Z zigbee-herdsman:adapter:ezsp:uart <-- [1243a1a9542a1fb049e6d5647e]
2023-03-02T00:49:57.129Z zigbee-herdsman:adapter:ezsp:uart <-- DATA (1,2,0): 1243a1a9542a1fb049e6d5647e
2023-03-02T00:49:57.130Z zigbee-herdsman:adapter:ezsp:uart --> ACK  (2)
2023-03-02T00:49:57.130Z zigbee-herdsman:adapter:ezsp:uart --> [82503a7e]
2023-03-02T00:49:57.131Z zigbee-herdsman:adapter:ezsp:ezsp <== Frame: 01800100000a021072
2023-03-02T00:49:57.132Z zigbee-herdsman:adapter:ezsp:ezsp <== 0x0: {"_id_":0,"_cls_":"version","_isRequest_":false,"protocolVersion":10,"stackType":2,"stackVersion":29200}
2023-03-02T00:49:57.132Z zigbee-herdsman:adapter:ezsp:uart -+- waiting (2) success
2023-03-02T00:49:57.134Z zigbee-herdsman:adapter:ezsp:ezsp Set EzspConfigId.CONFIG_FRAGMENT_DELAY_MS = 50
2023-03-02T00:49:57.135Z zigbee-herdsman:adapter:ezsp:ezsp ==> setConfigurationValue: {"configId":29,"value":50}
2023-03-02T00:49:57.135Z zigbee-herdsman:adapter:ezsp:ezsp ==> {"_cls_":"setConfigurationValue","_id_":83,"_isRequest_":true,"configId":29,"value":50}
2023-03-02T00:49:57.136Z zigbee-herdsman:adapter:ezsp:uart --> DATA (2,2,0): 02000153001d3200
2023-03-02T00:49:57.136Z zigbee-herdsman:adapter:ezsp:uart --> [224021a9072a088059b1b07e]
2023-03-02T00:49:57.138Z zigbee-herdsman:adapter:ezsp:uart -?- waiting (3)
2023-03-02T00:49:57.138Z zigbee-herdsman:adapter:ezsp:uart <-- [2340a1a9072a159ab77e]
2023-03-02T00:49:57.139Z zigbee-herdsman:adapter:ezsp:uart <-- DATA (2,3,0): 2340a1a9072a159ab77e
2023-03-02T00:49:57.139Z zigbee-herdsman:adapter:ezsp:uart --> ACK  (3)
2023-03-02T00:49:57.140Z zigbee-herdsman:adapter:ezsp:uart --> [83401b7e]
2023-03-02T00:49:57.141Z zigbee-herdsman:adapter:ezsp:ezsp <== Frame: 028001530000
2023-03-02T00:49:57.142Z zigbee-herdsman:adapter:ezsp:ezsp <== 0x53: {"_id_":83,"_cls_":"setConfigurationValue","_isRequest_":false,"status":0}
2023-03-02T00:49:57.142Z zigbee-herdsman:adapter:ezsp:uart -+- waiting (3) success
2023-03-02T00:49:57.143Z zigbee-herdsman:adapter:ezsp:ezsp Set EzspConfigId.CONFIG_FRAGMENT_WINDOW_SIZE = 1
2023-03-02T00:49:57.143Z zigbee-herdsman:adapter:ezsp:ezsp ==> setConfigurationValue: {"configId":28,"value":1}
2023-03-02T00:49:57.143Z zigbee-herdsman:adapter:ezsp:ezsp ==> {"_cls_":"setConfigurationValue","_id_":83,"_isRequest_":true,"configId":28,"value":1}
2023-03-02T00:49:57.144Z zigbee-herdsman:adapter:ezsp:uart --> DATA (3,3,0): 03000153001c0100
2023-03-02T00:49:57.145Z zigbee-herdsman:adapter:ezsp:uart --> [334121a9072a09b35929487e]
2023-03-02T00:49:57.146Z zigbee-herdsman:adapter:ezsp:uart -?- waiting (4)
2023-03-02T00:49:57.147Z zigbee-herdsman:adapter:ezsp:uart <-- [3441a1a9072a15f1287e]
2023-03-02T00:49:57.147Z zigbee-herdsman:adapter:ezsp:uart <-- DATA (3,4,0): 3441a1a9072a15f1287e
2023-03-02T00:49:57.147Z zigbee-herdsman:adapter:ezsp:uart --> ACK  (4)
2023-03-02T00:49:57.148Z zigbee-herdsman:adapter:ezsp:uart --> [8430fc7e]
2023-03-02T00:49:57.148Z zigbee-herdsman:adapter:ezsp:ezsp <== Frame: 038001530000
2023-03-02T00:49:57.149Z zigbee-herdsman:adapter:ezsp:ezsp <== 0x53: {"_id_":83,"_cls_":"setConfigurationValue","_isRequest_":false,"status":0}
2023-03-02T00:49:57.150Z zigbee-herdsman:adapter:ezsp:uart -+- waiting (4) success
2023-03-02T00:49:57.151Z zigbee-herdsman:adapter:ezsp:ezsp Set EzspConfigId.CONFIG_INDIRECT_TRANSMISSION_TIMEOUT = 30000
2023-03-02T00:49:57.151Z zigbee-herdsman:adapter:ezsp:ezsp ==> setConfigurationValue: {"configId":18,"value":30000}
2023-03-02T00:49:57.151Z zigbee-herdsman:adapter:ezsp:ezsp ==> {"_cls_":"setConfigurationValue","_id_":83,"_isRequest_":true,"configId":18,"value":30000}
2023-03-02T00:49:57.152Z zigbee-herdsman:adapter:ezsp:uart --> DATA (4,4,0): 0400015300123075
2023-03-02T00:49:57.153Z zigbee-herdsman:adapter:ezsp:uart --> [444621a9072a07822cf7b77e]
2023-03-02T00:49:57.154Z zigbee-herdsman:adapter:ezsp:uart -?- waiting (5)
2023-03-02T00:49:57.203Z zigbee-herdsman:adapter:ezsp:uart <-- [4546a1a9072a1505697e]
2023-03-02T00:49:57.204Z zigbee-herdsman:adapter:ezsp:uart <-- DATA (4,5,0): 4546a1a9072a1505697e
2023-03-02T00:49:57.204Z zigbee-herdsman:adapter:ezsp:uart --> ACK  (5)
2023-03-02T00:49:57.205Z zigbee-herdsman:adapter:ezsp:uart --> [8520dd7e]
2023-03-02T00:49:57.206Z zigbee-herdsman:adapter:ezsp:ezsp <== Frame: 048001530000
2023-03-02T00:49:57.206Z zigbee-herdsman:adapter:ezsp:ezsp <== 0x53: {"_id_":83,"_cls_":"setConfigurationValue","_isRequest_":false,"status":0}
2023-03-02T00:49:57.206Z zigbee-herdsman:adapter:ezsp:uart -+- waiting (5) success
2023-03-02T00:49:57.207Z zigbee-herdsman:adapter:ezsp:ezsp Set EzspConfigId.CONFIG_SECURITY_LEVEL = 5
2023-03-02T00:49:57.207Z zigbee-herdsman:adapter:ezsp:ezsp ==> setConfigurationValue: {"configId":13,"value":5}
2023-03-02T00:49:57.208Z zigbee-herdsman:adapter:ezsp:ezsp ==> {"_cls_":"setConfigurationValue","_id_":83,"_isRequest_":true,"configId":13,"value":5}
2023-03-02T00:49:57.208Z zigbee-herdsman:adapter:ezsp:uart --> DATA (5,5,0): 05000153000d0500
2023-03-02T00:49:57.208Z zigbee-herdsman:adapter:ezsp:uart --> [554721a9072a7d38b759b3b97e]
2023-03-02T00:49:57.209Z zigbee-herdsman:adapter:ezsp:uart -?- waiting (6)
2023-03-02T00:49:57.210Z zigbee-herdsman:adapter:ezsp:uart <-- [5647a1a9072a15af307e]
2023-03-02T00:49:57.211Z zigbee-herdsman:adapter:ezsp:uart <-- DATA (5,6,0): 5647a1a9072a15af307e
2023-03-02T00:49:57.211Z zigbee-herdsman:adapter:ezsp:uart --> ACK  (6)
2023-03-02T00:49:57.212Z zigbee-herdsman:adapter:ezsp:uart --> [8610be7e]
2023-03-02T00:49:57.212Z zigbee-herdsman:adapter:ezsp:ezsp <== Frame: 058001530000
2023-03-02T00:49:57.213Z zigbee-herdsman:adapter:ezsp:ezsp <== 0x53: {"_id_":83,"_cls_":"setConfigurationValue","_isRequest_":false,"status":0}
2023-03-02T00:49:57.213Z zigbee-herdsman:adapter:ezsp:uart -+- waiting (6) success
2023-03-02T00:49:57.214Z zigbee-herdsman:adapter:ezsp:ezsp Set EzspConfigId.CONFIG_END_DEVICE_POLL_TIMEOUT = 14
2023-03-02T00:49:57.214Z zigbee-herdsman:adapter:ezsp:ezsp ==> setConfigurationValue: {"configId":19,"value":14}
2023-03-02T00:49:57.215Z zigbee-herdsman:adapter:ezsp:ezsp ==> {"_cls_":"setConfigurationValue","_id_":83,"_isRequest_":true,"configId":19,"value":14}
2023-03-02T00:49:57.215Z zigbee-herdsman:adapter:ezsp:uart --> DATA (6,6,0): 0600015300130e00
2023-03-02T00:49:57.216Z zigbee-herdsman:adapter:ezsp:uart --> [664421a9072a06bc5926127e]
2023-03-02T00:49:57.217Z zigbee-herdsman:adapter:ezsp:uart -?- waiting (7)
2023-03-02T00:49:57.218Z zigbee-herdsman:adapter:ezsp:uart <-- [6744a1a9072a15803c7e]
2023-03-02T00:49:57.218Z zigbee-herdsman:adapter:ezsp:uart <-- DATA (6,7,0): 6744a1a9072a15803c7e
2023-03-02T00:49:57.219Z zigbee-herdsman:adapter:ezsp:uart --> ACK  (7)
2023-03-02T00:49:57.219Z zigbee-herdsman:adapter:ezsp:uart --> [87009f7e]
2023-03-02T00:49:57.220Z zigbee-herdsman:adapter:ezsp:ezsp <== Frame: 068001530000
2023-03-02T00:49:57.221Z zigbee-herdsman:adapter:ezsp:ezsp <== 0x53: {"_id_":83,"_cls_":"setConfigurationValue","_isRequest_":false,"status":0}
2023-03-02T00:49:57.222Z zigbee-herdsman:adapter:ezsp:uart -+- waiting (7) success
2023-03-02T00:49:57.222Z zigbee-herdsman:adapter:ezsp:ezsp Set EzspConfigId.CONFIG_STACK_PROFILE = 2
2023-03-02T00:49:57.222Z zigbee-herdsman:adapter:ezsp:ezsp ==> setConfigurationValue: {"configId":12,"value":2}
2023-03-02T00:49:57.223Z zigbee-herdsman:adapter:ezsp:ezsp ==> {"_cls_":"setConfigurationValue","_id_":83,"_isRequest_":true,"configId":12,"value":2}
2023-03-02T00:49:57.223Z zigbee-herdsman:adapter:ezsp:uart --> DATA (7,7,0): 07000153000c0200
2023-03-02T00:49:57.224Z zigbee-herdsman:adapter:ezsp:uart --> [774521a9072a19b059f3237e]
2023-03-02T00:49:57.225Z zigbee-herdsman:adapter:ezsp:uart -?- waiting (0)
2023-03-02T00:49:57.225Z zigbee-herdsman:adapter:ezsp:uart <-- [7045a1a9072a15eba37e]
2023-03-02T00:49:57.226Z zigbee-herdsman:adapter:ezsp:uart <-- DATA (7,0,0): 7045a1a9072a15eba37e
2023-03-02T00:49:57.226Z zigbee-herdsman:adapter:ezsp:uart --> ACK  (0)
2023-03-02T00:49:57.227Z zigbee-herdsman:adapter:ezsp:uart --> [8070787e]
2023-03-02T00:49:57.228Z zigbee-herdsman:adapter:ezsp:ezsp <== Frame: 078001530000
2023-03-02T00:49:57.228Z zigbee-herdsman:adapter:ezsp:ezsp <== 0x53: {"_id_":83,"_cls_":"setConfigurationValue","_isRequest_":false,"status":0}
2023-03-02T00:49:57.229Z zigbee-herdsman:adapter:ezsp:uart -+- waiting (0) success
2023-03-02T00:49:57.229Z zigbee-herdsman:adapter:ezsp:ezsp Set EzspConfigId.CONFIG_PACKET_BUFFER_COUNT = 255
2023-03-02T00:49:57.230Z zigbee-herdsman:adapter:ezsp:ezsp ==> setConfigurationValue: {"configId":1,"value":255}
2023-03-02T00:49:57.230Z zigbee-herdsman:adapter:ezsp:ezsp ==> {"_cls_":"setConfigurationValue","_id_":83,"_isRequest_":true,"configId":1,"value":255}
2023-03-02T00:49:57.231Z zigbee-herdsman:adapter:ezsp:uart --> DATA (0,0,0): 080001530001ff00
2023-03-02T00:49:57.236Z zigbee-herdsman:adapter:ezsp:uart --> [004a21a9072a144d59175d7e]
....chopped off....
```

**Comment by paulovitin** [comment]
Same issue here...

**Comment by tommyjlong** [comment]
This has now been fixed in 1.30.3-1 and has been working a couple of weeks. So closing
